apps/async-worker/async_worker/services/file_id_consumer.py

```
@@ -0,0 +1,101 @@
+            result = await r.brpop(FILEID_QUEUE_KEY, timeout=0)
+            if result is None:
+                continue
+
+            _, raw_payload = result
+            payload = json.loads(raw_payload)
+            await _process_file_id_update(payload)
+
+        except asyncio.CancelledError:
+            logger.info("file_id consumer cancelled, shutting down")
+            break
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)